plugins/grpc-plugin/src/main.rs

```diff
@@ -34,18 +34,22 @@ async fn main() -> Result<()> {
             options::Value::Integer(-1),
             "Which port should the grpc plugin listen for incoming connections?",
         ))
-        .start()
+        .configure()
         .await?;
 
     let bind_port = match plugin.option("grpc-port") {
         Some(options::Value::Integer(-1)) => {
             log::info!("`grpc-port` option is not configured, exiting.");
+            plugin.disable("`grpc-port` option is not configured.").await?;
```

Comment:
So, if I'm understanding the API, this `disable` method gives you the possibility to change the method init.
Does this mean if in the future core lightning will need another parameter in the response, we need another method?

or a chain of methods because we need to join method payloads in only one result, right?